client/js/libs/jquery/stickyscroll.js

```diff
@@ -37,7 +37,7 @@ import jQuery from "jquery";
 				lastStick = Date.now();
 				this.scrollTop = this.scrollHeight;
 			})
-			.on("msg.sticky", keepToBottom)
+			.on("keepToBottom.sticky", keepToBottom)
```

Comment:
Gotta trigger "keep to bottom" logic, but can't call the already used `msg` event.